packages/@aws-cdk/aws-ec2/lib/peer.ts

```diff
@@ -112,6 +122,16 @@ class CidrIPv6 implements IPeer {
   public readonly uniqueId: string;
 
   constructor(private readonly cidrIpv6: string) {
+    const cidrMatch = cidrIpv6.match(/^([\da-f]{0,4}:){2,7}([\da-f]{0,4})?(\/\d+)?$/);
```

Comment:
I used some fairly loose RegExps, especially for IPv6. The stricter ones tend to become a little crazy